docs/docs/secret/scanning.md

```diff
@@ -111,6 +106,13 @@ All rules are disabled except for the ones you specify, so it runs very fast.
 On the other hand, you should use `disable-rules` if you just want to disable some built-in rules.
 See the [enable-rules][enable-rules] and [disable-rules][disable-rules] sections for the detail.
 
+If you don't need secret scanning, you can disable it via the `--security-checks` flag.
+
+```shell
+$ trivy image --security-checks vuln alpine:3.15
```

Comment:
this option will skip `config` too. is it right?

Reply:
Misconfiguration detection cannot be used with image scanning for now.

Reply:
oh, ok, i missed it